tac/governing-documents/defining-adopters.md

```diff
@@ -0,0 +1,52 @@
+
+* **Direct adopters** are responsible for actively developing, packaging, configuring, or deploying the LFDT project as part of their use.
+
+* **Transitive adopters** benefit from the project indirectly but are not responsible for its integration or operation. However, if a transitive adopter gains the ability to configure or extend the project due to the service provider’s implementation, they may become direct adopters as well.
+
+The intent of identifying adopters within the LFDT community is to gain insight into real-world, production-level usage of its projects or specifications. This helps assess project maturity, community engagement, and long-term viability within the decentralized trust ecosystem.
+
+In some cases, original sponsoring organizations or maintainers may also be recognized as adopters. The LFDT and the associated LF/DT projects’ technical and community leadership reserves the discretion to determine whether an adopter can offer meaningful feedback to guide the evolution of a project.
+
+Adopters of LFDT projects may include the following (noting that a single organization may fit more than one category):
+
+* **Ecosystem Participant** – An organization actively engaged in developing, deploying, or operating one or more LFDT projects.
+
+* **Service Provider** – An entity offering LFDT projects as part of a larger solution (e.g., a platform or product).
+
+* **Upstream or Downstream Project** – An open source project that depends on or integrates with an LFDT project for functionality, compatibility, or supply chain inclusion.
+
+* **End-user Organization** – A company or institution that uses an LFDT project to power part of its infrastructure, services, or internal workflows, without necessarily contributing upstream.
+
+When documenting adopters (e.g., in an `ADOPTERS.md` file), projects are encouraged to identify the category or categories each adopter fits under, but to list the adopter only once. This helps the LFDT community track usage patterns and adoption trends more effectively.
+
+* End users \- companies and organizations that are not LFDT End-User members that use the project and decentralized trust technologies internally, or build upon a decentralized trust open source project but do not sell the decentralized trust project externally as a service offering. This group is identified in the written form by the convention end user, uncapitalized and unhyphenated.  
+* A Service Provider \- a Service Provider is an organization that repackages an open source project as a core component of a service offering and sells decentralized trust services externally. Examples might include:  
+  * APIs  
+  * SaaS  
+  * blockchain networks both public and private~~cloud providers~~.   
```

Comment:
Repackaged supported software might be an another example?

Reply:
Added.